tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -474,3 +474,28 @@ Given('the user has created the following files', function (entryList) {
   })
   return client
 })
+
+When('the user copies permalink to the current folder', function () {
+  return client.page.filesPage().copyPermalink()
+})
+Then('the clipboard content should match current folder permalink', function () {
+  return client.execute(function () {
```

Comment:
would be good to put it all into a separate function, e.g. in custom commands, so it can be reused

Reply:
done